internal-docs/custom-themes.md

```diff
@@ -31,13 +31,13 @@ class MyThemeContext extends DefaultThemeRenderContext {
         const site = this.options.getValue("gaId");
 
         const script = `
-(function() {
-    var _owa = document.createElement('script'); _owa.type = 'text/javascript';
-    _owa.async = true; _owa.src = '${site}' + '/modules/base/js/owa.tracker-combined-min.js';
-    var _owa_s = document.getElementsByTagName('script')[0]; _owa_s.parentNode.insertBefore(_owa,
-    _owa_s);
-}());
-`.trim();
+            (function() {
```

Comment:
The indentation here was intentional, the extra whitespace added here will be included in the output, which likely isn't what is desired. Perhaps moving the `const script` outside of the class so that it is naturally non-indented would be better?

Reply:
As suggested, I moved the `const script` outside the class. Not sure if the name is suitable.

Reply:
Moved the script outside of class